src/GovUk.Education.ExploreEducationStatistics.Admin/Areas/Identity/Pages/Account/ExternalLogin.cshtml.cs

```diff
@@ -238,6 +242,13 @@ private IActionResult RedirectToLoginPageWithError(string returnUrl)
             ExternalLoginInfo info,
             string returnUrl)
         {
+            var inviteExpiryTime = inviteToSystem.Created + InviteExpirySpan;
+
+            if (DateTime.UtcNow >= inviteExpiryTime)
+            {
+                return await HandleExpiredInvite(inviteToSystem, email);
+            }
```

Comment:
Yeah I think we should bring this up with Cam tomorrow.  As far as I understood, this ticket was actually around expiring invites to the service itself rather than adding users to entities after that.

Reply:
I think maybe my comment isn't correct anyway - I've got a feeling that if an existing user is invited to have a release/publication permission, then they are instantly given that permission, I think.